### PR TITLE
Bilingual Tag Display

### LINKED ISSUE #154: Bilingual Topics management

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #154 
*Please link to the related issue, using the [references] keyword only if the PR completes the entire issue*

**Proposed Changes**
Better Support for Topics in Japanese language

*Description of changes in this PR*
- [x] Update Airtable to add a `name_ja` column for topics and add translations, removed some duplicates
- [x] Use the locale to toggle between locales for tags
- [x] Update the nomination form to show bilingual tags for selection, while preserving the ability to submit. (I wonder if this will work with autocomplete...)

**Testing Plan**
1) Browse through the speakers and verify that tags appear. Toggle locale to Japanese and ensure that most tags appear in Japanese, and some fall back to English where translations are absent.
2) Nominate a speaker. Add some existing tags using
   a)  the checkbox.
   b) type completion
   c) a custom submission by typing into the input.

### CODE REVIEW

File: web/src/views/FindSpeaker.vue [hunk 1]
```diff
@@ -134,7 +134,7 @@ export default {
       return lastPageEntry > this.speakers.length ? this.speakers.length : lastPageEntry;
     },
   },
-  mounted() {
+  async mounted() {
```
Comment (tuttiq) [inline web/src/views/FindSpeaker.vue:137]: Why this change?

**Comment by tuttiq** [comment]
The code looks good to me, but about this

> Update Airtable to add a name_ja column for topics and add translations, removed some duplicates

Did you update the speaker references to the topics before you removed the duplicates? I just wanna make sure we didn't  accidentally remove some topics linked to speakers without replacing with a translated one.

**Review by tuttiq — APPROVED** [review]
Code LGTM

File: web/src/views/FindSpeaker.vue (hunk 1 above)
Comment (ann-kilzer) [reply web/src/views/FindSpeaker.vue:137]: Oops, forgot to remove this. Good catch.

**Comment by ann-kilzer** [comment]
> The code looks good to me, but about this
> 
> > Update Airtable to add a name_ja column for topics and add translations, removed some duplicates
> 
> Did you update the speaker references to the topics before you removed the duplicates? I just wanna make sure we didn't accidentally remove some topics linked to speakers without replacing with a translated one.

Yes, I retained all the proper relationships while removing duplicates.
